sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamReader.scala

```diff
@@ -535,5 +536,5 @@ final class DataStreamReader private[sql](sparkSession: SparkSession) extends Lo
 
   private var userSpecifiedSchema: Option[StructType] = None
 
-  private var extraOptions = new scala.collection.mutable.HashMap[String, String]
+  private var extraOptions = CaseInsensitiveMap[String](Map.empty)
```

Comment:
`DataStreamReader` support reading DS v2, shall we get the original map at that place?

Reply:
Thanks. It's updated now.